source/index.html.md

```diff
@@ -175,7 +175,7 @@ error categories, the last two digits define specific errors.
 * 0345: Quote is expired or invalid
 * 0346: You need to agree to the new terms and condition
 * 0347: Too many attempts to validate CURP
-* 0348: CURP data did not match with user data
+* 0348: KYC data did not match with user data
```

Comment:
while this is correct, but not sure if the API users know what is KYC. 
```suggestion
* 0348: KYC data did not match with user data. (CURP-México, X-Argentina, X-Brasil)
```

makes sense?

Reply:
Agreed, change is now in place.